response_operations_ui/views/messages.py

```diff
@@ -104,27 +97,33 @@ def view_conversation(thread_id):
 
         try:
             message_controllers.send_message(
-                _get_message_json(form,
-                                  thread_id=refined_thread[0]['thread_id'])
+                _get_message_json(form, thread_id=refined_thread[0]['thread_id'])
             )
             thread_url = url_for("messages_bp.view_conversation", thread_id=thread_id, page=page,
                                  conversation_tab=conversation_tab) + "#latest-message"
             flash(Markup(f'Message sent. <a href={thread_url}>View Message</a>'))
             return redirect(url_for('messages_bp.view_selected_survey', selected_survey=refined_thread[0]['survey'],
                                     page=page, conversation_tab=conversation_tab))
 
-        except (ApiError, InternalError):
+        except (ApiError, InternalError) as e:
```

Comment:
This section may read better if it had separate except blocks and a finally instead of  looking at the error type ?

Reply:
Removed class check, but couldn't get it looking right